documentation/leo/additional_material/00_tooling.md

```diff
@@ -28,19 +28,19 @@ Follow these steps to toggle the `Leo` syntax highlighting, hover, and tokens.
 1. Open `Sublime Text`.
 2. From Preferences > Select Color Scheme... > LSP-aleo-developer
 
-## VSCode
+## VS Code
 
 [//]: # (![]&#40;./images/vscode.png&#41;)
 Download the editor here: https://code.visualstudio.com/download.
 
 ### Install
 
-Install [VSCode for Leo](https://marketplace.visualstudio.com/items?itemName=aleohq.leo-extension) from VSCode marketplace.   
+Install [VS Code for Leo](https://marketplace.visualstudio.com/items?itemName=aleohq.leo-extension) from VSCode marketplace.   
```

Comment:
Maybe this should say 'VS Code plugin for Leo'?

Reply:
Looked for an external reference and found 

> "C/C++ for Visual Studio Code"
> https://code.visualstudio.com/docs/languages/cpp

Reply:
Maybe we should use that pattern then.